Controllers/Services/EmployeeReconciliation/EmployeeReconciliationService.cs

```diff
@@ -313,56 +347,61 @@ private async Task<Employee> ReconcileWithDatabase(Employee employee)
             }
         }
 
-
-        public async Task<Employee> UpdateEmployeeStatus(
-            Employee employee
-        )
+        private async Task<Employee> UpdateEmployeeStatus(Employee employee)
         {
-            var callWebStatusCode = await callWeb
-                .GetSurveyStatusCode(employee);
+            var callWebStatusCode = await callWeb.GetSurveyStatusCode(employee);
+
+            if (callWebStatusCode == null)
+            {
+                // TODO: Fix this
```

Comment:
Fix what?

Reply:
Thanks for the catch. In the latest pushed code, now throws an exception, as this is an unexpected condition.